assets/views/layout.ejs

```diff
@@ -64,7 +64,7 @@
     </div>
 
     <!--SCRIPTS-->
-    <script id="app" src="/js/app.js" <% if (typeof _csrf !== 'undefined') { %> _csrf="<%= _csrf %>" <% } %>></script>
+    <script id="app" src="<%= sails.config.environment === 'development' ? '/js/app.js' : '/min/production.min.js' %>" <% if (typeof _csrf !== 'undefined') { %> _csrf="<%= _csrf %>" <% } %>></script>
```

Comment:
As a side note, we should probably stick a version number on the end of this at some point. I thought we had already done that, but ah well. Will help with some cache issues.